src/semver/Version.d

```diff
@@ -64,6 +64,24 @@ struct Version
         );
     }
 
+    /**
+        Returns:
+            string representation of the version stripped from
+            and metadata information
+     **/
+    string toStringNoMetadata ( ) const pure
```

Comment:
wouldn't it be nicer to make  `toString` use this function and let it just append the metadata?